iele-gas.md

```diff
@@ -828,22 +845,39 @@ Note: These are all functions as the operator `#compute` has already loaded all
       requires L1 >=Int L2
     
     rule Cdiv(SCHED, L1, L2) =>
-        Gdivword < SCHED > *Int L1 +Int //TODO(remove this term)
+        Gdivword < SCHED > *Int L1 +Int
+        Gdiv < SCHED >
+      requires notBool Gnewarith << SCHED >>
+      [owise]
+
+    rule Cdiv(SCHED, L1, L2) =>
```

Comment:
Why doesn't the new div cost depend on the size of the numbers at all?

Reply:
Because we measured it and it doesn't depend on them in this case; this is the case when you divide a smaller number by a larger one and get zero.

Reply:
whoops, looks like I messed up.

Reply:
ah, no, this is correct. note that this is only the owise case when L1 <Int L2

Reply:
Ah, the diff context didn't show the case above.